app-admin/prelude-manager/prelude-manager-3.1.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools systemd
+
+DESCRIPTION="Bus communication for all Prelude modules"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="tcpwrapper xml geoip dbx"
+
+RDEPEND="net-libs/gnutls:=
+	dev-libs/libprelude
```

Comment:
Per https://github.com/Prelude-SIEM/prelude-manager/blob/master/configure.ac#L50 we need `>=dev-libs/libprelude-3.1.0`